src/core/main.c

```diff
@@ -50,10 +53,13 @@ int main(int argc, char *argv[]) {
 
 		switch (c) {
 		case 'l':
-			loglevel = get_loglevel(optarg);
+			config.logger_conf.loglevel = get_loglevel(optarg);
 			break;
 		case 'o':
-			output_name = optarg;
+			config.screencast_conf.output_name = optarg;
```

Comment:
We need to `strdup` here.

Reply:
So there wasn't a strdup used before, because the output option would accidentality be used last?